h/services/bulk_api/lms_stats.py

```diff
@@ -17,65 +17,83 @@ class AssignmentStats:
     last_activity: datetime
 
 
+@dataclass
+class CourseStats:
```

Comment:
The diff is not showing this to me right now but instead of `CourseStats" this is now "CountsByAssignment"